docs/os-deps.md

```diff
@@ -0,0 +1,83 @@
+# OS dependencies
+
+k0s is packaged as a fully statically linked binary which contains all other needed components also as statically linked binaries. This means that in typical use cases there's no OS level dependencies.
```

Comment:
The first sentence seems a bit complex. Could it be simplified in some way, for example:
"k0s is packaged as a single binary, which includes all the needed components as statically linked."